docs/Testing/OSXWinObjCTests.md

```diff
@@ -0,0 +1,172 @@
+# WinObjC Unit Tests on OSX
+
+As a measure to help ensure the accuracy of the WinObjC platform, we are beginning to support building and running WinObjC unit tests on OSX devices. By verifying our tests against the reference platform, we hope to reduce errors in test writing and incorrect assumptions about Objective-C APIs. Foundation.UnitTests are currently supported, with other test modules expected to follow suit by the end of July.
```

Comment:
I'll just cut out exactly this part?

---
In reply to: [70347028](https://github.com/Microsoft/WinObjC/pull/634#discussion_r70347028) [](ancestors = 70347028)